### DISCUSSION TITLE
New results: October 15th

### DISCUSSION THREAD

**Discussion by lascavana** [ds4dm/ml4co-competition#100] [Announcements]
Dear participants,

The new results are out !
https://www.ecole.ai/2021/ml4co-competition/#leaderboard

For this new evaluation round, we have lowered the memory usage limit of SCIP from 15GiB to 12GiB. This unfortunately did not solve the OOM crash issue of one of the teams. However, we keep it as a safety measure. It did not have a significant impact on the rankings.

Also, please remember that we are now using Ecole version 0.7.3, so we kindly ask you to update your conda.yaml file accordingly.

As usual, if you notice something weird or unexpected in the leaderboard, please feel free to reach out to us.

Good luck, and have fun with your submissions !

Best,
The organizing team

**Comment by akangw** [comment]
Hi @lascavana,

Since it is near the submission deadline, I'm wondering whether it is possible to update the leaderboard more frequently. Thanks!

Best,
Vincent.
